keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -758,6 +753,21 @@ _It is expected that the `.clusterset.local` zone is standard and available in
 all implementations, but customization and/or aliasing can be explored if there's
 demand._
 
+##### Not allowing cluster-specific targeting via DNS
+
+Both ClusterSetIP Services and Multicluster Headless Services are specified to explicitly disallow DNS records that target single clusters, such as using a formulation like `<clusterid>.<svc>.<ns>.svc.clusterset.local.` to target all 1+N backends in a single cluster.
```

Comment:
So what we propose here, is that we can still access individual pods under a headless service (L736), but not service-wide to a specific cluster.

Is that correct?

Reply:
Correct, and I think you found in your later comment the rationale section behind that decision. You also might be interested to check out the recording from [SIG-Multicluster's 3/16/2021 meeting starting 36:28](https://youtu.be/6mqE8MDiSnQ?t=2188) where we talked about it.

Reply:
Do we need to say that this name structure is "reserved for possible future use" or that it is a MUST NOT?

Reply:
Fixed in 8c25fa3